src/main/java/com/artipie/SliceFromConfig.java

```diff
@@ -167,7 +169,7 @@ static Slice build(
                 break;
             case "npm":
                 slice = new NpmSlice(
-                    cfg.path(),
+                    new Unchecked<>(() -> new URL(cfg.path())).value(),
```

Comment:
@olenagerasimova you should pass `cfg.url()` as first parameter to `Slice`. Also to fix the proof you need to change `path: "/npm_repo/"` to `url: http://localhost:8080/npm_repo` in `npm_repo.yaml`